apiserver/facades/schema.json

```diff
@@ -17702,7 +17702,7 @@
                         "SerialId": {
                             "type": "string"
                         },
-                        "Size": {
+                        "SizeMiB": {
```

Comment:
Was this intentional?

If possible we should try and keep the old ones for compatibility reasons.

Reply:
Good pickup, this was unintentional. Whoever did this work originally messed up by not using a bespoke params struct for BlockDevice. I'll need to add one.